mac

```diff
@@ -116,13 +116,10 @@ fancy_echo 'Checking on Node.js installation...'
 
 if ! brew_is_installed "node"; then
   if ! command -v nvm > /dev/null; then
-    if ! command -v n > /dev/null; then
-      fancy_echo 'Installing n and latest Node.js and NPM...'
-      curl -L http://git.io/n-install | bash -s -- -y latest
-    else
-      fancy_echo 'Updating n...'
-      n-update -y
```

Comment:
Like, this should still be removed. I think. We're encouraging people to use `nvm` by installing it, and talking about it in the guide. But we shouldn't have two node versioning managers running on the user's system in case they already installed `n`.